app/src/main/java/com/github/se/eventradar/ui/qrCode/QrCodeTicketUi.kt

```diff
@@ -221,6 +220,17 @@ fun QrCodeTicketUi(
             item { Spacer(modifier = Modifier.height(8.dp)) }
           }
     } else {
+      EventTitle(
```

Comment:
personally do not agree wiht the UI choice here, does not make sense in my personal opinion to have the event title here, and I think makes the UI less nice. But these are simply UI preferenecs so if u wanna keep it like that, so be it :)